### PR TITLE
Update 'More than search' pdf & add to related resources

### LINKED ISSUE #5611: Update "More than search" PDF  and add to related resources
Stakeholder: Lori Colston
What? Update the existing PDF and make it more discoverable within the applicable, related resources sections
Why? Make our differentiators more discoverable.

Assets:
Existing PDF: https://about.sourcegraph.com/guides/key-traits-of-a-code-intelligence-platform.pdf
New PDF: [Key traits of a code intelligence platform.pdf](https://github.com/sourcegraph/about/files/9209645/Key.traits.of.a.code.intelligence.platform.pdf)

**Update these pages with the PDF in the Related Resources section:**
https://about.sourcegraph.com/use-cases/code-health ( Replace the third resource - How we added backend integration testing to our CI pipeline)
https://about.sourcegraph.com/use-cases/onboarding (add a third related resource)
https://about.sourcegraph.com/code-insights ( replace the second resource - Announce Code Insight) This one has the least amount of clicks according to Crazy Egg cc [@Alex Isken](https://app.asana.com/0/1200584296872142/list)
https://about.sourcegraph.com/use-cases/code-reuse (add a third resource)

**Copy for new Related resource entry:**
5 key traits of a code intelligence platform
PDF
Sourcegraph is more than search. Learn how the code intelligence platform helps development teams quickly get unblocked, resolve issues faster, and gain insights to make better decisions. 

Asana: https://app.asana.com/0/1200095757619647/1202669989163394/f

### PR DESCRIPTION
**Pull request by st0nebraker**
Closes #5611. Updates 'More than search' pdf & adds it to **Related Resources** section on select pages.

### Changelog
""

### Test
1. Ensure prettier has standardized the proposed changes.
2. Ensure **Related Resources** are updated accordingly in changes files.

### CODE REVIEW

File: src/pages/code-insights.tsx [hunk 1]
```diff
@@ -271,14 +271,15 @@ const blogResourceItems = [
...
-        description: 'Learn about why we built Code Insights from our CEO.',
-        type: 'Blog post',
+        title: '5 key traits of a code intelligence platform',
+        description:
+            'Sourcegraph is more than search. Learn how the code intelligence platform helps development teams quickly get unblocked, resolve issues faster, and gain insights to make better decisions.',
+        type: 'Guide',
         img: {
-            src: 'https://storage.googleapis.com/sourcegraph-assets/blog/code-insights-ga-blogs/announcement-header.png',
-            alt: 'Announcing code insights blog thumbnail',
+            src: '/backgrounds/dark-multi-grid.jpg',
```
Comment (bretthayes) [inline src/pages/code-insights.tsx:279]: I was going to suggest that we change the src to be `string | ReactNode` to allow us to use our Background component here with the `darkMultiGrid` variant, but I want to improve the way we use backgrounds that now that we have Tailwind going. It's currently a WIP in my branch. I'd also like to move away from asset-heavy backgrounds and use more native code to generate interesting backgrounds. I'm currently working with @traceyljohnson to improve this.

The current Background component also assumes that all backgrounds are hero backgrounds so they're quite large assets to load for places where thumbnails would be more suitable. The only variants that have mobile and tablet versions are the `darkNebulous*` variants.

Maybe for now you can export a thumbnail version and place it in `/public/blog/thumbnails` or `/public/bg-images/thumbnails`. What do you think?

**Review by bretthayes — CHANGES_REQUESTED** [review]
Everything looks good! I just have a thought about our backgrounds/thumbnails below:

File: src/pages/code-insights.tsx (hunk 1 above)
Comment (st0nebraker) [reply src/pages/code-insights.tsx]: Great points. And yes! Super interested to invest more in native code generated bg's 😄 

I was curious about our bg asset plan for some other scenarios like this in my draft PR right now. I updated the file path for this asset temporarily! Let me know if something else should change

Comment (bretthayes) [reply src/pages/code-insights.tsx]: Sounds good! Path looks good for now!

**Review by bretthayes — APPROVED** [review]

**Pull request merged**